pkg/securitypolicy/securitypolicy.go

```diff
@@ -115,6 +116,20 @@ type IDName struct {
 	Name string
 }
 
+func MeasureSeccompProfile(seccomp *specs.LinuxSeccomp) (string, error) {
```

Comment:
just to be explicit, "measure" means the sha256 hash as a hex-string?

Reply:
Yes.